instances/latest/terminologies/technique/phaseContrastMicroscopy.jsonld

```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/phaseContrastMicroscopy",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": null,
```

Comment:
Try to adapt a definition from wikipedia entry: https://en.wikipedia.org/wiki/Phase-contrast_microscopy

Reply:
done